### DISCUSSION TITLE
"Read More" jumps to cart

### DISCUSSION THREAD

**Discussion by choskins-godaddy** [bootscore/bootscore#358] [Q&A] [answered]
I have some out of stock products and, instead of "Add to Cart", those items say, "Read More". When clicking "Read More", it used to go to the product page but it starting going to the side cart (I think it might be called "mini cart" but not sure). However, if they click the item itself FROM THE product-category page (not the main page because it uses bs-swiper) and not the "Read More" part, it takes the user to the correct place.

I'm wondering if someone could point me in the right direction.

**Comment by crftwrk** [comment, marked as answer]
This is fixed here https://github.com/bootscore/bootscore/pull/329. Release is next Monday but you can download main branch as well if you do not want to wait.

**Reply by choskins-godaddy** [reply]
When you say next Monday, do you mean the 16th or do you mean the 23rd?

**Reply by crftwrk** [reply]
Today is Saturday, next Monday means the day after tomorrow ;-)
